<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeMenu</class>
 <widget class="QMainWindow" name="EmployeeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="productTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>561</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="ownerFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="emailInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Контактный телефон</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Контактная почта</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Склад</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация о пользователе</string>
     </property>
    </widget>
    <widget class="QComboBox" name="warehouseInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="welcomeLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataUpdateButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
